### DISCUSSION TITLE
Problema Accesso HA da APP

### DISCUSSION THREAD

**Discussion by omarantelmi** [indomus/forum#1580] [HUB: Home Assistant]
Chiedo il vostro aiuto se possibile 

Non riesco ad accedere a Home Assistant tramite l'app IOS e browser, o meglio, non riesco ad accedere con la rete dati cellulare.
In wifi locale si connette, se mi connetto su altri wifi non locali, si connette
Se provo a fare Hotspot dal mio cellulare a pc e poi ad accedere da browser o app si connette, se vado con la rete dati non si connette e va in timeout

Ho provato anche a configurare un nuovo utente, collegare un nuovo cellulare tramite rete dati di quel cellulare e funziona tutto

Non ho attivato nessun servizio di BAN (almeno non mi risulta)
Ho disabilitato l'acceso a due fattori
Sono su raspberri pi4 + docker ha + docker reverse proxy + duckdns configurato in home assistant

Allego il log di Caddy perchè credo che ci sia qualche errore da qualche parte, ma non riesco a capire quale sia

[LOG ELIMINATO]

**Comment by indomus** [comment]
che provider internet hai

**Comment by omarantelmi** [comment]
Vodafone sia a casa che su cellulare

**Reply by indomus** [reply]
sei sotto NAT o hai IP pubblico

**Reply by indomus** [reply]
(non aprire cento risposte, rispondi qui sotto)

**Reply by omarantelmi** [reply]
Si scusami
No non sono sotto NAT ne ho IP PUBBLICO, ma anche se fosse non si spiegherebbe per quale motivo solo il mio cellulare non riesce ad accedere
Utilizzo duckdns per aggiornare l'ip che cambia

**Reply by snakuzzo** [reply]
Innanzi tutto quando posti il log dovresti assicurarti di nascondere i tuoi dati (nel file che hai linkato c'è tutto in chiaro...ip...fqdn...porte...etc).
Ad ogni modo, io non penso sia legato al tuo problema. Nell'errore ci sono `connection refused`...tu invece dici che va in timeout, quindi se va in timeout sul server di casa nemmeno ci arrivi.

Sicuro di non utilizzare una APN particolare (tipo aziendale) ?

**Reply by omarantelmi** [reply]
La faccio breve
Ho ripristinato il cellulare, e ho risolto
Forse qualche file rimasto attivo anche dopo le varie disinstallazioni dell'app per cercare di risolvere il problema
Confermo la risoluzione al momento
